src/components/Recipe.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Card, Modal } from "react-bootstrap";
+import "../styles/Recipe.css";
+
+class Recipe extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modal: false,
+    };
+  }
+
+  setModal = (flag) => {
+    this.setState({modal: flag});
+  }
+
+  displayIngredients = () => {
+    let ingredList = [];
+    const {ingredients} = this.props.data;
+    for (let i in ingredients) {
+      ingredList.push(<li key={i}>{ingredients[i]}</li>);
+    }
+    return ingredList;
+  }
+
+  displayProcedure = () => {
+    let procList = [];
+    const {method} = this.props.data;
+    for (let i in method) {
+      procList.push(<li key={i}>{method[i]}</li>);
+    }
+    return procList;
+  }
+
+  // TODO (issue 26): Card/Modal image
+  render() {
+    return (
+      <div className="container">
+        <div className="rounded-border d-inline p-1 bg-light">
+          <Card border="dark">
+            <Card.Img variant="top" src=""/>
+            <Card.Title>
+              <button className="card-title" onClick={() => this.setModal(true)}>
+                <strong>{this.props.data.name}</strong>
+              </button>
+            </Card.Title>
+            <Card.Body>
+              <div>Time: {this.props.data.time} min</div>
```

Comment:
Is your data.time always in minutes? What if it's 4 hours? Will it say 240 min instead?